app/Controllers/feedController.php

```diff
@@ -916,12 +916,15 @@ public function contentSelectorPreviewAction() {
 			return;
 		}
 
+		$attributes = $feed->attributes();
+		$attributes['path_entries_filter'] = trim(Minz_Request::param('selector_filter'));
```

Comment:
```suggestion
		$attributes['path_entries_filter'] = trim(Minz_Request::param('selector_filter', ''));
```

trim() does not accept (anymore, PHP8.1+) a null, which is the default value